Documentation/contributing/development/reviewers_committers/duties.rst

```diff
@@ -0,0 +1,49 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _weekly_duties:
+
+Weekly Duties
```

Comment:
Sentence case in headings? 🥺  
```suggestion
Weekly Duties
```

Reply:
OK, but then let's make it “official”. I'll add a commit to change the directive in the style guide.